e2e/framework/config.go

```diff
@@ -46,6 +46,11 @@ type TestServerConfig struct {
 	ShowsLog                bool                 // Flag specifying if logs are shown
 	IsPos                   bool                 // Specifies the mechanism used for IBFT (PoA / PoS)
 	Signer                  *crypto.EIP155Signer // Signer used for transactions
+	//
+	UseBridge                    bool
+	BridgeRootChainURL           *string
+	BridgeRootChainContract      *string
+	BridgeRootChainConfirmations *uint
```

Comment:
Please document these settings better, as others (that are even more obvious) are documented 🙏

Reply:
Added at c7181b25e9fc6621f7ca89c10e118f0216f71709